src/eko/matching_conditions/operator_matrix_element.py

```diff
@@ -4,24 +4,67 @@
 |VFNS| evolution.
 """
 
+import functools
 import logging
-import time
 
 import numba as nb
 import numpy as np
 from scipy import integrate
 
 from .. import basis_rotation as br
-from .. import interpolation, mellin
 from ..anomalous_dimensions import harmonics
+from ..evolution_operator import Operator, QuadKerBase
 from ..member import OpMember
-from . import nlo, nnlo
+from . import n3lo, nlo, nnlo
```

Comment:
`import eko` was 19 min for me just now - maybe we should delay this line? Now, I'm quite convinced that we're abusing numba in some way - it should be JIT as promised ... Reading [here](https://numba.pydata.org/numba-doc/dev/reference/jit-compilation.html) and if I remember correctly it seems related to the signature ... Someone remembers why we have signatures everywhere? (I think otherwise cache doesn't work?) and why can't we run in `nopython` mode and would that help us?

Reply:
Indeed, I still believe we have signatures because of cache, but in general specifying signatures is like declaring types in C, it should not be harmful.

The actual problem is: why caching yields AOT?
I'd like to compile JIT, and once compiled caching it, but not before.

Reply:
okay, we need a better trick than the simple import statement delay... 
I think we are running in `nopython ` when we use `njit`, no?

Reply:
Yes, `nb.njit` should be just an alias for `nb.jit(nopython=True)`